host-interaction/mutex/create-locked-file.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: create locked file
```

Comment:
does this create a new locked file or lock an existing file? if its the latter, perhaps consider renaming to "lock file".